src/FakeItEasy/Configuration/PropertySetterConfiguration.cs

```diff
@@ -101,10 +78,8 @@ public IPropertySetterConfiguration WhenArgumentsMatch(Func<ArgumentCollection,
         }
 
         private static IPropertySetterConfiguration AsPropertySetterConfiguration(
-            IVoidConfiguration voidArgumentValidationConfiguration)
-        {
-            return new ProperySetterAdapter(voidArgumentValidationConfiguration);
-        }
+            IVoidConfiguration voidArgumentValidationConfiguration) =>
+            new ProperySetterAdapter(voidArgumentValidationConfiguration);

```

Comment:
I like it much better, although I suspect we'll have to fight Visual Studio auto-formatting to keep it that way...